spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -474,61 +391,335 @@ definitions:
+    fields:
+        - header:
+            type: GriddedCorrectionHeader
+            desc: Header of a gridded correction message
+        - element:
+            type: GridElement
+            desc: |
+              Tropo and STEC residuals for the given grid point.
+
+ - MSG_SSR_TILE_DEFINITION:
+    id: 0x05F6
+    short_desc: |
+      Definition of a SSR atmospheric correction tile.
+    desc: |
+      Provides the correction point coordinates for the atmospheric correction
+      values in the MSG_SSR_STEC_CORRECTION and MSG_SSR_GRIDDED_CORRECTION
+      messages.
+
+      Based on ETSI TS 137 355 V16.1.0 (LTE Positioning Protocol) information
+      element GNSS-SSR-CorrectionPoints. SBP only supports gridded arrays of
+      correction points, not lists of points.
+    fields:
+        - tile_set_id:
+            type: u8
+            desc: Unique identifier of the tile set this tile belongs to.
+        - tile_id:
+            type: u8
+            desc: |
+              Unique identifier of this tile in the tile set.
+
+              See GNSS-SSR-CorrectionPoints field correctionPointSetID.
+        - corner_nw_lat:
+            type: u16
+            units: encoded degrees
+            desc: |
+              North-West corner correction point latitude.
+
+              The relation between the latitude X in the range [-90, 90] and
+              the coded number N is:
+
+              N = floor((X / 90) * 2^14)
+
+              See GNSS-SSR-CorrectionPoints field referencePointLatitude.
+        - corner_nw_lon:
+            type: u16
+            units: encoded degrees
+            desc: |
+              North-West corner correction point longtitude.
+
+              The relation between the longtitude X in the range [-180, 180]
+              and the coded number N is:
+
+              N = floor((X / 180) * 2^15)
+
+              See GNSS-SSR-CorrectionPoints field referencePointLongitude.
+        - spacing_lat:
+            type: u16
+            units: 0.01 degrees
+            desc: |
+              Spacing of the correction points in the latitude direction.
+
+              See GNSS-SSR-CorrectionPoints field stepOfLatitude.
+        - spacing_lon:
+            type: u16
+            units: 0.01 degrees
+            desc: |
+              Spacing of the correction points in the longtitude direction.
+
+              See GNSS-SSR-CorrectionPoints field stepOfLongtitude.
+        - rows:
+            type: u16
+            desc: |
+              Number of steps in the latitude direction.
+
+              See GNSS-SSR-CorrectionPoints field numberOfStepsLatitude.
+        - cols:
+            type: u16
+            desc: |
+              Number of steps in the longtitude direction.
+
+              See GNSS-SSR-CorrectionPoints field numberOfStepsLongtitude.
+        - bitmask_valid:
+            type: u8
+            desc: |
+              If 1 then bitmask indicates valid correction points, otherwise if
```

Comment:
@ljbade so this is like a `boolean` on whether to use the bitmask?

Reply:
yeah I was trying to emulate what is in LPP, where the `bitmask` is an optional value...

alternatively I could copy what we do in SIP and remove this but invert the logic of bitmask, e.g. 0 = valid, 1 = invalid

Reply:
decided its easier to match SIP for the moment